client/solve.go

```diff
@@ -35,7 +35,8 @@ import (
 
 type SolveOpt struct {
 	Exports               []ExportEntry
-	LocalDirs             map[string]string
+	LocalDirs             map[string]string // Deprecated: use LocalFiles
+	LocalFiles            map[string]fsutil.FS
```

Comment:
I think `LocalPaths` or `LocalMounts` might be better names. `LocalPaths` is more like an alias for `LocalDirs` that isn't directory or file specific and `LocalMounts` might make more sense because each map entry is a definition of a local filesystem that can be mounted.

Reply:
`LocalMounts` SGTM, I'm happy with that.